src/plugins/imagery/components/Compass/Compass.vue

```diff
@@ -23,7 +23,7 @@
 <template>
 <div
     class="c-compass"
-    :style="compassDimensionsStyle"
+    :style="`width: ${ sizedImageDimensions.width }px; height: ${ sizedImageDimensions.height }px`"
```

Comment:
I think this is fine for now, but if we anticipate more styles, we may want another computed "compassStyles" property that returns these formatted values, similar to the old compassDimensionsStyle() computed value. I like the rename and having it available for other purposes, so I think just adding another computed variable just for the styles would work.

Reply:
No change requested here (?) so not addressing right now. If we need to control more styles on this in the future, we can address then.